### PR TITLE
chore: add expected output to the sample code.

### PR DESCRIPTION
**Pull request by sayakpaul**
Fixes the issue as described here: https://github.com/huggingface/transformers/pull/18815#issuecomment-1256338065

@ydshieh FYI.

### CODE REVIEW

File: src/transformers/models/vit_msn/modeling_vit_msn.py [hunk 1]
```diff
@@ -632,6 +632,8 @@ def forward(
         >>> from PIL import Image
         >>> import requests
 
+        >>> torch.manual_seed(2)
```
Comment (sayakpaul) [inline src/transformers/models/vit_msn/modeling_vit_msn.py:635]: Since the classification head of ViT MSN is randomly initialized we need this for a consistent output (`print(model.config.id2label[predicted_label])`).

File: src/transformers/models/vit_msn/modeling_vit_msn.py [hunk 2]
```diff
@@ -644,6 +646,7 @@ def forward(
         >>> # model predicts one of the 1000 ImageNet classes
         >>> predicted_label = logits.argmax(-1).item()
         >>> print(model.config.id2label[predicted_label])
+        LABEL_183
```
Comment (ydshieh) [inline src/transformers/models/vit_msn/modeling_vit_msn.py:649]: Thank you @sayakpaul . Just wondering, if there is a plan to update the config on the Hub for this checkpoint, then update this value here?

Comment (sayakpaul) [reply src/transformers/models/vit_msn/modeling_vit_msn.py:649]: How the config should be updated? Updated with the ImageNet-1k labels?

Comment (ydshieh) [reply src/transformers/models/vit_msn/modeling_vit_msn.py:649]: Sorry @sayakpaul , I should ask this question internally, as it is a checkpoint from fackebook `facebook/vit-msn-small`.
But yes, I think it should be updated with ImageNet-1k labels.

Comment (sayakpaul) [reply src/transformers/models/vit_msn/modeling_vit_msn.py:649]: Sure. When the classification head is updated with the pretrained params, I think we could revisit it.

Comment (ydshieh) [reply src/transformers/models/vit_msn/modeling_vit_msn.py:649]: Or maybe you would like to open a Hub PR for that checkpoint?

Comment (sayakpaul) [reply src/transformers/models/vit_msn/modeling_vit_msn.py:649]: Yeah that is what my plan is i.e., open a PR to the MSN checkpoints on Hub to update the config.

**Comment by sayakpaul** [comment]
@ydshieh here's what I have done so far:

* Update the weight conversion script, so we have the ImageNet-1k label mapping in the model config. 
* Update the `modeling_vit_msn.py` script with the expected output (which is basically an ImageNet-1k class).

I also pushed the model classes with the updated configs. I realized if I only updated the `config.json` in the respective Hub repositories, the member config associated with the model classes may not get appropriately updated. So, I pushed everything updated with the new config to my HF account. 

So, if you go to https://huggingface.co/models?filter=vit_msn now, you'd notice two versions of the same model checkpoint: one is hosted from `facebook` and another one is hosted from `sayakpaul`. I need someone from the HF team to transfer the updated model repositories to `facebook`. 

Let me know if anything here is unclear.

**Review by ydshieh — APPROVED** [review]
Thank you @sayakpaul , nice we hvae labels now :-)

**Comment by ydshieh** [comment]
> I realized if I only updated the config.json in the respective Hub repositories, the member config associated with the model classes may not get appropriately updated.

Could you explain a bit more about `the member config associated with the model classes may not get appropriately updated.
`?

**Comment by sayakpaul** [comment]
The current model classes living under the Facebook org don't have the updated configs that have the ImageNet-1k labels. 

The model classes residing under `sayakpaul` do. They need to be transferred to the facebook org. 

If you search with https://huggingface.co/models?filter=vit_msn then you will get the complete list. 

Let me know if anything is unclear now.

**Comment by ydshieh** [comment]
My question is about why uploading the new config.json is not enough - this is what I understand from your comments.

**Comment by sayakpaul** [comment]
Okay, let me verify a few things and get back to you.

**Review by sgugger — APPROVED** [review]
LGTM for the changes in Transformers! For configs already on the Hub, a PR to update them will be needed indeed.
Thanks!

**Comment by sayakpaul** [comment]
@ydshieh @sgugger the PRs to the Hub repos are up:

* https://huggingface.co/facebook/vit-msn-small/discussions/4
* https://huggingface.co/facebook/vit-msn-base/discussions/2
* https://huggingface.co/facebook/vit-msn-large/discussions/2
* https://huggingface.co/facebook/vit-msn-base-4/discussions/2
* https://huggingface.co/facebook/vit-msn-large-7/discussions/2

Indeed, just updating the config.json made it work. I have verified it locally.

**Comment by ydshieh** [comment]
Hub PRs all merged, thank you @sayakpaul (also for checking)
